5_recognition.ipynb

```diff
@@ -430,60 +775,109 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "`lego`は、絶対座標上で約（x, y）=（0.6, 0.0）の座標に置かれているようです。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "`lego` seems to be placed at about (x, y) = (0.6, 0.0) position on absolute coordinates."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "## 認識結果を用いた制御"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Control using recognition results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "以上の認識器を用いて、HSRにレゴブロックを拾わせてみましょう。\n",
     "\n",
     "前章の関数`move_wholebody_ik`に認識結果のx, yを引数として与え、逆運動学によってHSRを制御します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's make HSR pick up the Lego block using the above recognizer.\n",
+    "\n",
+    "The function `move_wholebody_ik` in the previous section is given the recognition result x and y as arguments, and the HSR is controlled by inverse kinematics:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ハンドを開く\n",
+    "# ハンドを開く Open the hand\n",
     "move_hand(1)\n",
-    "# 逆運動学を使ってHSRの手先を動かします\n",
+    "# 逆運動学を使ってHSRの手先を動かします Move the end effector by using inverse kinematics\n",
     "move_wholebody_ik(x, y, 0.1, 180, 0, 0)\n",
-    "# ハンドを閉じる\n",
+    "# ハンドを閉じる Close the hand\n",
     "move_hand(0)\n",
-    "# アームを初期姿勢に戻す\n",
+    "# アームを初期姿勢に戻す Move the hand to the initial position\n",
     "move_arm_init()"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>課題</b>\n",
     "    <p>レゴブロックを出現させる位置を変えても上手く拾えるか試してみましょう。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Confirm that the picking will succeed even if we change the position of the Lego block.</p>\n",
```

Comment:
This sentence makes me think there is one specific known Lego block being referred to here.

    "    <p>Check that the picking up the Lego block will succeed even if we change its position.</p>\n",

Reply:
We can only detect "light blue" Lego block.

Reply:
So, the sentence should look like this:

   "    <p>Check that the picking up the light blue Lego block will succeed even if we change its position.</p>\n",